Dbal/PaymentAttributes.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdyenPayment\Dbal;
+
+use Enlight_Components_Db_Adapter_Pdo_Mysql;
+
+class PaymentAttributes
+{
+    /**
+     * @var Enlight_Components_Db_Adapter_Pdo_Mysql
+     */
+    private $db;
+
+    public function __construct(Enlight_Components_Db_Adapter_Pdo_Mysql $db)
+    {
+        $this->db = $db;
+    }
+
+    public function fetchByAdyenType(string $adyenType): array
+    {
+        $paymentAttributesResult = $this->db
```

Comment:
This query fetches all data, but the implementation is only using the `paymentmeanID`

Two options here: 
1. refactor method name and code to only return the `paymentmeanID`
2. when more data is needed, use the ORM (thus class needs to be moved from Dbal to ORM and return the ORM Entity